cves/kernel/CVE-2013-3224.yml

```diff
@@ -253,8 +270,8 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: This was a flaw in safely handling micro systems related to the Bluetooth protocol in the linux kernel and does not seem to involve il8n in any way.
```

Comment:
Small, but it should be 'i18n' not 'il8n'

Reply:
updated. thanks!